src/teleop/gui/src/components/MoteusStateTable.vue

```diff
@@ -1,101 +1,189 @@
+            </th>
+            <th class="tableElement">
+              {{ moteusStateData.error[indices.FrontRight] }}
+            </th>
+          </tr>
+          <tr>
+            <th class="tableElement tableHeader">Name</th>
+            <th class="tableElement tableHeader">
+              {{ moteusStateData.name[indices.MiddleLeft] }}
+            </th>
+            <th class="tableElement tableHeader">
+              {{ moteusStateData.name[indices.MiddleRight] }}
+            </th>
+          </tr>
+          <tr>
+            <th class="tableElement tableHeader">State</th>
+            <th class="tableElement">
+              {{ moteusStateData.state[indices.MiddleLeft] }}
+            </th>
+            <th class="tableElement">
+              {{ moteusStateData.state[indices.MiddleRight] }}
+            </th>
+          </tr>
+          <tr>
+            <th class="tableElement tableHeader">Error</th>
+            <th class="tableElement">
+              {{ moteusStateData.error[indices.MiddleLeft] }}
+            </th>
+            <th class="tableElement">
+              {{ moteusStateData.error[indices.MiddleRight] }}
+            </th>
+          </tr>
+          <tr>
+            <th class="tableElement tableHeader">Name</th>
+            <th class="tableElement tableHeader">
+              {{ moteusStateData.name[indices.BackLeft] }}
+            </th>
+            <th class="tableElement tableHeader">
+              {{ moteusStateData.name[indices.BackRight] }}
+            </th>
+          </tr>
+          <tr>
+            <th class="tableElement tableHeader">State</th>
+            <th class="tableElement">
+              {{ moteusStateData.state[indices.BackLeft] }}
+            </th>
+            <th class="tableElement">
+              {{ moteusStateData.state[indices.BackRight] }}
+            </th>
+          </tr>
+          <tr>
+            <th class="tableElement tableHeader">Error</th>
+            <th class="tableElement">
+              {{ moteusStateData.error[indices.BackLeft] }}
+            </th>
+            <th class="tableElement">
+              {{ moteusStateData.error[indices.BackRight] }}
+            </th>
+          </tr>
+        </thead>
+      </table>
+    </div>
+
+    <div v-else>
+      <div>
+        <h3>{{ header }}</h3>
+      </div>
+      <table
+        class="tableFormat"
+        style="undefined;table-layout: fixed; width: 800px"
+      >
+        <colgroup>
+          <col style="width: 20px" />
+          <col style="width: 30px" />
+          <col style="width: 30px" />
+          <col style="width: 30px" />
+          <col style="width: 30px" />
+          <col style="width: 30px" />
+          <col style="width: 30px" />
+        </colgroup>
+        <thead>
+          <tr>
+            <th class="tableElement tableHeader">Name</th>
+            <th class="tableElement tableHeader">
+              {{ moteusStateData.name[indices.FrontLeft] }}
```

Comment:
Why are indices.FrontLeft being used for the arm version? I think I need a bit more explanation on how these cases actually work.

Reply:
^ bump.